### PR TITLE
Add written intro after the infographic

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Added written version of the infographic

### CODE REVIEW

**Comment by dennisreimann** [comment]
Nice, This provides more context to the page and maybe a chunk of this could start before the inforgraphic to guide the user better. 

I spotted two typos: „different[e]“ anf „transaction[‘]s“.

**Comment by RiccardoMasutti** [comment]
> Nice, This provides more context to the page and maybe a chunk of this could start before the inforgraphic to guide the user better.
> 
> I spotted two typos: „different[e]“ anf „transaction[‘]s“.

Fixed.

File: docs/why-wasabi/README.md [hunk 1]
```diff
@@ -12,3 +12,33 @@ Here is covered how Bitcoin is already good for privacy, much better than the in
 And why Wasabi can make Bitcoin to an even better tool of self defense by default.
 
 ![](/InfographicWhyWasabi.png)
+
+## Gaining privacy with Wasabi Wallet for Bitcoin transactions
+
+### The peirceived problem
```
Comment (yahiheb) [inline docs/why-wasabi/README.md]:
```suggestion
### The perceived problem
```

File: docs/why-wasabi/README.md [hunk 2]
```diff
@@ -12,3 +12,33 @@ Here is covered how Bitcoin is already good for privacy, much better than the in
...
+
+Bitcoin transactions are somewhat *traceable* and this creates the need for a wallet that specializes in privacy.
+
+*_"What kind of privacy? I'm not sure if I need that."_*
+
+When we make traditional financial transactions (banks, PayPal, etc.), no one but the service provider is able to track our spendings habits or account balances.
```
Comment (yahiheb) [inline docs/why-wasabi/README.md]:
```suggestion
When we make traditional financial transactions (Banks, PayPal, etc.), no one but the service provider is able to track our spending habits or account balances.
```

File: docs/why-wasabi/README.md [hunk 3]
```diff
@@ -12,3 +12,33 @@ Here is covered how Bitcoin is already good for privacy, much better than the in
...
+
+### Wasabi Wallet
+
+Wasabi Wallet is an *open source* and *non-custodial* desktop wallet offered by zkSNACKs Ltd.
```
Comment (yahiheb) [inline docs/why-wasabi/README.md]:
```suggestion
Wasabi Wallet is an *open source*, *non-custodial*, *privacy-focused*, desktop Bitcoin wallet offered by zkSNACKs Ltd.
```

File: docs/why-wasabi/README.md [hunk 4]
```diff
@@ -12,3 +12,33 @@ Here is covered how Bitcoin is already good for privacy, much better than the in
...
+it differentiates itself from many other wallets for its strong *focus on user privacy*.
```
Comment (yahiheb) [inline docs/why-wasabi/README.md]:
```suggestion
It differentiates itself from many other wallets for its strong *focus on user privacy*.
```

**Pull request merged**